packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -142,14 +146,14 @@ Set a rich text value using one of the Lexical formatting APIs such as [`toggleF
 
 ### With Chat Log
 
-Use Chat Composer alongside other Paste components such as [Chat Log](/components/chat-log) to build more complex chat UI. This example also makes use of a Lexical
-plugin `AutoScrollPlugin`.
+Use Chat Composer alongside other Paste components such as [Chat Log](/components/chat-log) to build more complex chat UI.
 
 <LivePreview
```

Comment:
Extra shaking going on in this example

Uploading Screen Recording 2023-08-18 at 2.13.42 PM.mov…

Reply:
That's kind of the problem with the animation. It needs reworking a fair bit

Reply:
it is because the scrollbar is going from hidden to shown to hidden